pkg/controllers/servicesync/servicesync.go

```diff
@@ -219,11 +219,17 @@ func (e *ServiceSyncer) syncServiceAndEndpoints(ctx context.Context, fromService
 				},
 			},
 			Spec: corev1.ServiceSpec{
-				Ports:     fromService.Spec.Ports,
-				ClusterIP: corev1.ClusterIPNone,
+				Ports: fromService.Spec.Ports,
 			},
 		}
 
+		// Check if ClusterIP is not "None"
+		if fromService.Spec.ClusterIP != corev1.ClusterIPNone {
+			toService.Spec.ClusterIP = fromService.Spec.ClusterIP
+		} else {
+			toService.Spec.ClusterIP = corev1.ClusterIPNone
+		}
```

Comment:
I haven't had a moment to see if this is the correct fix for the issue, but since this conditional sets to service to none if from service is none, or the value of from service it should be the same as 
```suggestion
				Ports: fromService.Spec.Ports,
				ClusterIP: fromService.Spec.ClusterIP,
			},
		}
```